### PR TITLE
Add class parameter `fpm_log_dir_mode` to customize the file permission of log directory

### PR DESCRIPTION
**Pull request by Q-Storm**
#### Pull Request (PR) description

I created the PR to parametrize the file permission for the php-fpm log directory.

### CODE REVIEW

**Review by kenyon — CHANGES_REQUESTED** [review]
Would be nice to also add a test to https://github.com/voxpupuli/puppet-php/blob/master/spec/classes/php_spec.rb

**Comment by jnk0** [comment]
Also mentioned in #523.

**Comment by Q-Storm** [comment]
> Would be nice to also add a test to https://github.com/voxpupuli/puppet-php/blob/master/spec/classes/php_spec.rb

This is my first rspec test. I hope the test is as expected.

File: manifests/init.pp [hunk 1]
```diff
@@ -147,6 +147,7 @@
   Optional[String[1]] $fpm_package                = undef,
   String[1] $fpm_user                             = $php::params::fpm_user,
   String[1] $fpm_group                            = $php::params::fpm_group,
+  Pattern[/^\d+$/] $fpm_log_dir_mode              = $php::params::fpm_log_dir_mode,
```
[the remark this answers is not in the document]
Comment (Q-Storm) [reply manifests/init.pp]: I implemented the changes a suggested. Nice to know that stdlibs covers this.

**Review by kenyon — APPROVED** [review]

**Review by smortex — APPROVED** [review]

**Pull request merged**